plugins/nodes/src/js/pages/nodes/nodes-table/NodesTableContainer.js

```diff
@@ -142,7 +190,12 @@ class NodesTableContainer extends mixin(StoreMixin, QueryParamsMixin) {
         <DrainNodeModal
           open={selectedNodeToDrain != null}
           node={selectedNodeToDrain}
-          onClose={this.handleDrainClose}
+          onClose={this.handleCloseModal}
+        />
+        <DeactivateNodeConfirm
+          open={selectedNodeToDeactivate != null}
```

Comment:
Why "!="?

Reply:
I was lazy and didnt look up the behavior of null strict comparison